main.py

```diff
@@ -0,0 +1,47 @@
+def model_run(request):
```

Comment:
Great - so this should work for the --entry-point in GCP CLI.  @raphaeljafriLB have we validated this yet? https://cloud.google.com/functions/docs/deploying/filesystem

Reply:
havent validated this yet - once committed I can test to deploy the repo to gcloud